homeassistant/components/simplisafe/sensor.py

```diff
@@ -0,0 +1,80 @@
+"""Support for SimpliSafe freeze sensor."""
+from simplipy.entity import EntityTypes
+
+from homeassistant.const import DEVICE_CLASS_TEMPERATURE, TEMP_FAHRENHEIT
+from homeassistant.core import callback
+
+from . import SimpliSafeEntity
+from .binary_sensor import SimpliSafeSensorBattery
+from .const import DATA_CLIENT, DOMAIN
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up SimpliSafe freeze sensors based on a config entry."""
+    simplisafe = hass.data[DOMAIN][DATA_CLIENT][entry.entry_id]
+
+    async_add_entities(
+        [
+            SimplisafeFreezeSensor(simplisafe, system, sensor)
+            for system in simplisafe.systems.values()
+            for sensor in system.sensors.values()
+            if sensor.type == EntityTypes.temperature
+        ]
+    )
+
+    # Add low battery status entity for every sensor
+    async_add_entities(
+        [
+            SimpliSafeSensorBattery(simplisafe, system, sensor)
```

Comment:
The sensor battery enrity should have unit of measurement set to percentage.

Reply:
SimpliSafe just provides a flag saying if the battery is low or not, we don't get a percentage value

Reply:
Then we shouldn't add the battery sensor entity just the battery binary sensor entity.

Reply:
so, `SimpliSafeSensorBattery` actually is a binary sensor entity, not a sensor entity. Is this not correct?

Reply:
A `BinarySensorEntity` can only be created and added from a binary_sensor platform. What are we trying to accomplish?

Reply:
Ah ok I see what you mean. I'm trying to add a binary sensor entity with the battery status for every sensor. I'll amend so that they are all added within the binary_sensor platform.